datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -311,3 +311,26 @@ def get_tox_env_python_version(env):
     match = re.match(PYTHON_MAJOR_PATTERN, env)
     if match:
         return int(match.group(1))
+
+
+def process_checks_option(checks, source=None):
+    # provide common function for determining which checks to run validations against
+    # `source` determines which method for gathering valid checks, default will use `get_valid_checks`
+
+    if source is None or source == 'checks':
+        get_valid = get_valid_checks
+    elif source == 'metrics':
+        get_valid = get_metric_sources
+    elif source == 'testable':
+        get_valid = get_testable_checks
+    else:
+        get_valid = get_valid_integrations
```

Comment:
May be redundant but wdyt of having an explicit `elif source == 'integrations'` check that provided the same result as the else?
A lot of the callers of this method pass in `source='integrations'`, so it may be helpful to have this be an explicit option.